File: 3533-snake-in-matrix/snake-in-matrix.py
class Solution:
    def finalPositionOfSnake(self, n: int, commands: List[str]) -> int:

        answer = 0

        for command in commands:
            if command == 'RIGHT':
                answer += 1
            elif command == 'LEFT':
                answer -= 1
            elif command == 'DOWN':
                answer += n
            else:
                answer -= n

        return answer

            
        

File: 3533-snake-in-matrix/snake-in-matrix.java
class Solution {
    public int finalPositionOfSnake(int n, List<String> commands) {
        int answer = 0;

        for (String command : commands){
            if (command.equals("RIGHT")){
                answer += 1;
            }
            else if (command.equals("LEFT")){
                answer -= 1;
            }
            else if (command.equals("DOWN")){
                answer += n;
            }
            else {   // UP
                answer -= n;
            }
        }
        return answer;
    }
}
